baybe/constraints/discrete.py

```diff
@@ -42,6 +44,24 @@ def get_invalid(self, data: pd.DataFrame) -> pd.Index:  # noqa: D102
         res = reduce(_valid_logic_combiners[self.combiner], satisfied)
         return data.index[res]
 
+    def to_polars(self) -> pl.Expr:
+        """Translate the constraint to Polars expression for filtering.
+
+        Returns:
+            The Polars Expr object to pass as an argument to filter().
+        """
+        satisfied = []
+        for k, cond in enumerate(self.conditions):
+            satisfied.append(cond.to_polars(pl.col(self.parameters[k])))
+
+        # to prevent errors when we have empty condition list
```

Comment:
* Comment should be capitalized
* Put a new line between the assignment  and the next comment
* Can you elaborate on "to prevent errors ..."? I don't understand what is meant here

Reply:
@rjavadi  please clarify